runtimelib/tests/kernels/ir/kernel.json

```diff
@@ -0,0 +1,13 @@
+{
+  "argv": [
+    "/Library/Frameworks/R.framework/Resources/bin/R",
```

Comment:
With these checked in files you _might_ be able to allow these to be just a flat `R` as the first argument. Same for the execvr below.